File: management-ui/src/types/Service.ts
// Copyright © VNG Realisatie 2021
// Licensed under the EUPL
//

interface Service {
  organization: {
    serialNumber: string
    name: string
  }
  service: string
  serviceName: string
  state: string
  apiSpecificationType?: string
  latestAccessRequest?: Record<string, unknown>
  latestAccessProof?: Record<string, unknown>

  requestAccess(): void
  retryRequestAccess(): void
}

export default Service

File: directory-ui/domain/service.go
// Copyright © VNG Realisatie 2023
// Licensed under the EUPL

package domain

type Service struct {
	Name                     string
	OrganizationName         string
	OrganizationSerialNumber string
	IsOnline                 bool
	APISpecificationType     string
	PublicSupportContact     string
	DocumentationURL         string
}
